tests/test_utils/test_schemaloader.py

```diff
@@ -51,79 +51,64 @@ def test_imports_relative(input_path):
     ]
 
 
-@pytest.mark.skip(reason="Re-enable this once we get fully migrated")
-def test_error_paths(input_path):
-    """Test various loader error situations"""
-
-    fn = input_path("loadererror1.yaml")
-    with pytest.raises(ValueError, match="Unknown slot domain should fail") as e:
-        SchemaLoader(fn).resolve()
-    assert 'loadererror1.yaml", line 11, col 13' in str(e.value)
-
-    fn = input_path("loadererror2.yaml")
-    with pytest.raises(ValueError, match='type "string" does not declare a URI'):
-        SchemaLoader(fn).resolve()
-
-    fn = input_path("loadererror2a.yaml")
-    with pytest.raises(ValueError, match="slot: s1 - key and identifier slots cannot be optional"):
-        SchemaLoader(fn).resolve()
-
+@pytest.mark.parametrize(
+    ("file", "error"),
```

Comment:
The majority of these tests were in the same form -- load a dodgy file, check the error message. I parametrised them to save on repetition of the same code.